### PR TITLE
Todo list

### PR DESCRIPTION
**Pull request by MuriloRoque**
## The look of the User Interface

When you open the app, you will see three buttons
- New Project: You can create a new Project
- New Todo: You can create a new Todo and Associate a project to it
- Show Project: A list of all Projects is displayed and the todo for each project with the edit and delete buttons

### CODE REVIEW

File: src/app/todos.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    const project = document.getElementById('project').value;
+    const todo = new Todo(title, description, date, priority, project, false);
+
+    todos.push(todo);
+    localStorage.setItem('todos', JSON.stringify(todos));
+
+    window.location.reload();
+    return false;
+  };
+};
```
Comment (maelfosso) [inline src/app/todos.js]: Separate the logic from DOM manipulation.
- Remove the TODO class from there
- When removing a TODO for example, `values.splice` and `localStorage.setItem` are all logic. Kindly remove it from there
- Same thing for todoComplete, saving todo

File: src/app/projects.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    const project = new Project(inputValue.value);
+    projects.push(project);
+    document.getElementById('id-project-form').reset();
+    localStorage.setItem('projects', JSON.stringify(projects));
+    createOptions();
+    window.location.reload();
+    return false;
+  };
+};
+export default projectForm;
```
Comment (maelfosso) [inline src/app/projects.js]: Separate DOM manipulation from Logic. 
- Calling `projects.push` is Logic
- `localStorage.setItem` is Logic
The code between the two are DOM manipulation.  SEPARTE IT.

Kindly, read the comments left about TODO.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle:

Hello 

1. Please, as indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/projects_requirement.png), There should be `default` project.

2. Please Separate the DOM and the Logic as indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/modules_requirement.png)

3. Read the comments below

Kindly make the suggested changes and submit a new code review request
Thanks

Mael FOSSO

**Comment by MuriloRoque** [comment]
Hello, @maelfosso thanks for the review. Just to clarify, we have a default project called "All Todos", do you want us to rename it to "Default"?

**Comment by Theophile-Kango** [comment]
Dear TSE, here are some pictures that show the default Project when you first open the app
default-project
default-project-todo
We made the required changes

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @MuriloRoque @Theophile-Kango good job team, kudos for  making all that corrections, however when I try to edit a Todo a new todo is created also leaving the old one in place this is not the desired behaviour, please look into that  :slightly_smiling_face: 

Please make changes and **request for another review** :slightly_smiling_face: 

**RAPHAEL NORIODE** :mask:

**Comment by MuriloRoque** [comment]
Hello there, thank you for noticing that bug, it slipped! :sweat_smile: 
I am looking forward to learning how to use testing in Javascript, so we are able to notice the mistakes easily.
We just made the required changes!

**Review by bellom — APPROVED** [review]
## Hi Team 👏 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**